data/transport missions.txt

```diff
@@ -2280,7 +2280,7 @@ mission "Terraforming 9"
 	
 	on complete
 		dialog 
-			`The journey to Tundra was spent by your passengers discussing the possible methods to terraform the planet. The Republic representative was none too pleased when Nolan, perhaps jokingly, suggested that we crash an asteroid into the planet.` 
+			`The journey to Tundra was spent by your passengers discussing the possible methods of terraforming the planet. The Republic representative was none too pleased when Nolan, perhaps jokingly, suggested crashing an asteroid into it.`
```

Comment:
I agree with the changes to the crashing part of this line, although I don't think the terraforming part was necessary. The only thing that annoys me there is "the" before "possible methods".